packages/angular-material/src/other/table.renderer.ts

```diff
@@ -48,12 +48,33 @@ import {
 @Component({
   selector: 'TableRenderer',
   template: `
+
     <table
       mat-table
       [dataSource]="data"
       class="mat-elevation-z8"
       [trackBy]="trackElement"
     >
+      <ng-container matColumnDef="action">
+        <tr>
+          <th mat-header-cell *matHeaderCellDef>
+            <button mat-icon-button color="primary" (click)="add()" [disabled]="!isEnabled()" title="Add"><mat-icon>add</mat-icon></button>
```

Comment:
Use the material tooltip instead of the plain `title`. Use `mat-button` like for the delete buttons to have the same spacing
```suggestion
            <button mat-button color="primary" (click)="add()" [disabled]="!isEnabled()" matTooltip="Add"><mat-icon>add</mat-icon></button>
```

Reply:
Done